features/bootstrap/BehatRunnerContext.php

```diff
@@ -32,8 +27,12 @@ public function bootstrap()
         $this->workingDir = sprintf('%s/%s/', sys_get_temp_dir(), uniqid('BehatPageObjectExtension_'));
         $this->getFilesystem()->mkdir($this->workingDir, 0777);
 
-        $this->phpBin = $this->findPhpBinary();
-        $this->process = new Process(null);
+        $this->process = new Process([
+            $this->findPhpBinary(),
+            BEHAT_BIN_PATH,
+            strtr('--format-settings=\'{"timer": false}\'', array('\'' => '"', '"' => '\"')),
```

Comment:
@jakzal I think that `Process` takes care of this kind of escape somehow. Am I wrong?

Reply:
I remember this has been copied from the Behat test suite. If it can be simplified, I'm all for it!

Reply:
If I'm not missing something... https://github.com/symfony/process/blob/master/Process.php#L970
It should escape everything automatically, isn't it?

Reply:
If tests pass i'm happy with the change :)

Reply:
I'll try this evening or the next days.

Reply:
Ok, I've removed the escaping and seems to be fine.
Travis still failing due to the issue highlighted [here](https://github.com/sensiolabs/BehatPageObjectExtension/pull/128#issuecomment-559969374)

Reply:
👍